tools/sql_extraction/src/test/kotlin/DataFlowSolverTest.kt

```diff
@@ -39,13 +42,16 @@ class DataFlowSolverTest {
     }
 
     @Test
-    fun `solver throws without any valid frontends`() {
+    fun `solver returns empty sequence without any valid frontends`() {
         val frontEnd = mockk<FrontEnd>()
```

Comment:
Is it better practice to use mock annotation @MockK?

Reply:
I think it is more common in our codebase to use annotation and set up the 'shared' mock variable inside the test class, instead of creating them in each method.